var/spack/repos/builtin/packages/bamutil/package.py

```diff
@@ -14,23 +14,32 @@ class Bamutil(MakefilePackage):
 
     homepage = "https://genome.sph.umich.edu/wiki/BamUtil"
     url      = "https://genome.sph.umich.edu/w/images/7/70/BamUtilLibStatGen.1.0.13.tgz"
```

Comment:
Is this archive different from the one at https://github.com/statgen/bamUtil/archive/refs/tags/v1.0.13.tar.gz?